### PR TITLE
supply objc snippets for profiling setup

### PR DESCRIPTION
**Pull request by armcknight**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Polishing docs ahead of profiling GA release.

### CODE REVIEW

File: src/wizard/apple/profiling-onboarding/ios/2.configure-performance.md [hunk 1]
```diff
@@ -18,3 +18,13 @@ SentrySDK.start { options in
     options.tracesSampleRate = 1.0
 }
 ```
+
+```objc {tabTitle:Objective-C}
```
Comment (romtsn) [inline src/wizard/apple/profiling-onboarding/ios/2.configure-performance.md:22]: I think the wizard does not support tab titles, at least it used to be like that before :) but I guess you cannot test it without deploying

**Review by romtsn — APPROVED** [review]
lgtm except the wizard part 👍

File: src/wizard/apple/profiling-onboarding/ios/2.configure-performance.md (hunk 1 above)
Comment (armcknight) [reply src/wizard/apple/profiling-onboarding/ios/2.configure-performance.md:22]: Thanks for pointing that out! Yep, I see it in the wizard. I'll just make the change for the docs.

**Review by philipphofmann — APPROVED** [review]
LGTM

**Pull request merged**